package-res/resources/web/pentaho/visual/ccc/categoricalContinuousAbstract/View.js

```diff
@@ -67,6 +67,24 @@ define([
       this.base();
 
       this._configureAxisDisplayUnits(/* isPrimary: */true, "ortho");
+    },
+
+    _createChart: function(ChartClass, options) {
+
+      var chart = this.base(ChartClass, options);
+
+      var visualElemsCountMax = this._getVisualElementsCountMax();
+      if(visualElemsCountMax > 0) {
+        var me = this;
+        chart.override("_onWillCreatePlotPanelScene", function(plotPanel, data, axisSeriesDatas, axisCategDatas) {
```

Comment:
The method is called once for each plot panel that a chart has _before_ creating its scene tree. The name ain't perfect.
A chart can have more than one plot panel (e.g. column-line combo), although DET currently only exposes CCC charts with a single plot.